frontend/src/pages/Broadcast/Component/AllBroadcasts/AllBroadcasts.jsx

```diff
@@ -159,12 +162,31 @@ export function AllBroadcasts(props) {
               ListName="Filter by Year"
               ListItems={["2021", "2020"]}
```

Comment:
Can we please make the year list dynamic i.e starting from 2020 till current year?